databricks/koalas/tests/test_accessor.py

```diff
@@ -0,0 +1,122 @@
+import warnings
+import unittest
+import contextlib
+import numpy as np
+import pandas as pd
+import databricks.koalas as ks
+from databricks.koalas.testing.utils import assert_produces_warning
+from databricks.koalas.extensions import (
+    register_dataframe_accessor,
+    register_series_accessor,
+    register_index_accessor,
+)
```

Comment:
Could you reorganize imports?

We usually separate blocks based on the categories.

```py
import builtins

import general packages, numpy, pandas, pyspark, etc.

import koalas packages
```

Also, alphabetical ordering would be preferred.